Makefile

```diff
@@ -1,13 +1,16 @@
 PROG=ytfzf
 PREFIX=/usr/local
 BINDIR=${PREFIX}/bin
+DOCDIR=${PREFIX}/share/doc/ytfzf
 MANDIR=${PREFIX}/share/man
 
-man:
+doc:
 	mkdir -p ${DESTDIR}${MANDIR}/man1
 	mkdir -p ${DESTDIR}${MANDIR}/man5
+	mkdir -p ${DESTDIR}${DOCDIR}
 	install docs/man/ytfzf.1 ${DESTDIR}${MANDIR}/man1
 	install docs/man/ytfzf.5 ${DESTDIR}${MANDIR}/man5
+	install docs/conf.sh LICENSE ${DESTDIR}${DOCDIR}
```

Comment:
I should've mentioned this earlier, but `LICENSE` should go in `${PREFIX}/share/licenses/ytfzf`